build/common/openwisp/celery.py

```diff
@@ -3,16 +3,42 @@
 from celery import Celery
 from celery.schedules import crontab
 from openwisp.utils import env_bool
+from django.utils.timezone import timedelta
 
 os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'openwisp.settings')
 
-app = Celery('openwisp', include=['openwisp.tasks'])
-app.config_from_object('django.conf:settings', namespace='CELERY')
-app.autodiscover_tasks()
+radius_schedule, topology_schedule, monitoring_schedule = {}, {}, {}
+task_routes = {}
+
+if env_bool(os.environ.get('USE_OPENWISP_CELERY_NETWORK')):
+    task_routes['openwisp_controller.connection.tasks.*'] = {'queue': 'network'}
+
+if env_bool(os.environ.get('USE_OPENWISP_MONITORING')):
+    monitoring_schedule = {
+        'monitoring-periodic-task': {
```

Comment:
can we name this just `run_checks` for [consistency with ansible-openwisp2](https://github.com/openwisp/ansible-openwisp2/blob/master/templates/openwisp2/settings.py#L231)?

BTW, is this scheduled used anywhere at all?